site2/website/versioned_docs/version-2.3.0/reference-pulsar-admin.md

```diff
@@ -1296,7 +1296,7 @@ $ pulsar-admin namespaces set-schema-autoupdate-strategy tenant/namespace option
 Options
 |Flag|Description|Default|
 |----|---|---|
-|`-c`, `--compatibility`|Compatibility level required for new schemas created via a Producer. Possible values (Full, Backward, Forward).||
+|`-c`, `--compatibility`|Compatibility level required for new schemas created via a Producer. Possible values (Full, Backward, Forward, None).||
```

Comment:
remove this. since 2.3.0  has released and not include this type.

Reply:
👍 Nice catch ! @jiazhai 

@shiv4289 can you remove this from 2.3.0 doc?

Reply:
Ack